tajo-common/src/main/java/org/apache/tajo/SessionVars.java

```diff
@@ -145,7 +145,7 @@
 
   // for index
   INDEX_ENABLED(ConfVars.$INDEX_ENABLED, "index scan enabled", DEFAULT),
-  INDEX_SELECTIVITY_THRESHOLD(ConfVars.$INDEX_SELECTIVITY_THRESHOLD, "the selectivity threshold for index scan",
+  INDEX_SELECTIVITY_LIMIT(ConfVars.$INDEX_SELECTIVITY_LIMIT, "the selectivity limit for index scan",
```

Comment:
Why did you change the description as well? 
```threshold``` is more appropriate because it has a narrow meaning than ```limit```.